brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3014: Fix - Separator mobile width issue in frontend

### Description

[Separator] Separator width if set 20-30 in the tablet and if adding any width in the mobile view, the separator will not show up.
https://brainstormforce.atlassian.net/browse/SPCTR-3014

### Screenshots

https://d.pr/v/08j68H

### Types of changes





### How has this been tested?


- Add a Separator block on the new page.
- 
- Set the width of the separator block to 80% on the device.
- 
- Go to tablet view, and change the width to 20% - 30%.
- 
- Now change to mobile view, and update any random width of more than 50%.
- 
- Inspect the separator and see the width value.

### Checklist:
